app/globals.css

```diff
@@ -0,0 +1,19 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
+  --background: #ffffff;
+  --foreground: #171717;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #0a0a0a;
+    --foreground: #ededed;
+  }
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
- let's use `next/font` to manage fonts.
- [doc link](https://nextjs.org/docs/app/building-your-application/optimizing/fonts#with-tailwind-css)

Reply:
Fixed.